NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -6,16 +6,29 @@
 //
 
 #import "NewNoteViewController.h"
+#import "Tags.h"
+#import "TagsViewController.h"
+#import "Note.h"
+#import "TagCell.h"
+#import "TransferDelegate.h"
+#import "TextbookViewController.h"
+#import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
+#import "ImportDelegate.h"
+#import "CameraImport.h"
+#import "ErrorAlerts.h"
+
 
 @interface NewNoteViewController ()  <TransferDelegate, UIDocumentPickerDelegate, UIImagePickerControllerDelegate, UINavigationControllerDelegate, ImportDelegate>
-@property (strong, nonatomic) NSMutableArray *tags;
-@property (strong, nonatomic) NSMutableArray *books;
+@property (strong, nonatomic) NSMutableArray<Tags *> *tags;
+@property (strong, nonatomic) NSMutableArray<NSString *> *books;
 @property (strong, nonatomic) NSURL *selectedFileURL;
 @property (strong, nonatomic) IBOutlet UIButton *addNoteButton;
 @property (strong, nonatomic) UIMenu *importMenu;
 @property (strong, nonatomic) IBOutlet UIImageView *noteImageView;
 @property (strong, nonatomic) IBOutlet UITextField *titleTextField;
 @property (strong, nonatomic) IBOutlet UITextField *descriptionTextField;
+@property (nonatomic) CameraImport *cameraImporter;
```

Comment:
I assume you want this strongly held, we should be explicit like the other types